iesurveychecks/1-adapt-scto-forms.md

```diff
@@ -0,0 +1,28 @@
+
+# 1. Adapting SurveyCTO forms
+When [programming a questionnaire for electronic data collection](https://docs.surveycto.com/02-designing-forms/01-core-concepts/), it requires to be adapted to include the data quality checks by taking into account all possible scenarios that an enumerator may encounter during the collection process. (Instructions on how to create a SurveyCTO form can be found [here). When designing your survey form, it is important to take into account all possible scenarios that an enumerator may encounter during the collection process.
+
+### Prepare pre-loaded dataset
+
+The preloaded data should include at least the following items:
+[(see Preload Sample (without Module Completion))](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/iesurveychecks/scto/Preloaded%20Data%20Sample.xlsx))
+
+  1. Basic survey information such as the household/respondent id and assigned enumerator name.  Household/respondent ID should be unique
+  2. An attempt counter column with pre-assigned values of 0 (0 submissions). This variable describes the number of previous submissions for each survey
+  3. A completion status column with pre-assigned values of 0 (Incomplete). This variable will be changed to 1 once the survey is completed by the enumerator
```

Comment:
seems useful to specify here the variable name to use as the column title for both (2) and (3)